public/app/plugins/datasource/elasticsearch/components/QueryEditor/QueryTypeSelector.tsx

```diff
@@ -1,35 +1,21 @@
 import { SelectableValue } from '@grafana/data';
 import { RadioButtonGroup } from '@grafana/ui';
 
-import { MetricAggregation } from '../../dataquery.gen';
 import { useDispatch } from '../../hooks/useStatelessReducer';
+import { queryTypeToMetricType } from '../../queryDef';
 import { QueryType } from '../../types';
 
 import { useQuery } from './ElasticsearchQueryContext';
 import { changeMetricType } from './MetricAggregationsEditor/state/actions';
 import { metricAggregationConfig } from './MetricAggregationsEditor/utils';
 
-const OPTIONS: Array<SelectableValue<QueryType>> = [
+export const OPTIONS: Array<SelectableValue<QueryType>> = [
   { value: 'metrics', label: 'Metrics' },
   { value: 'logs', label: 'Logs' },
   { value: 'raw_data', label: 'Raw Data' },
   { value: 'raw_document', label: 'Raw Document' },
 ];
```

Comment:
Since we are exporting these, I think it makes sense to move these options to the utils.ts as well